build.sbt

```diff
@@ -22,7 +22,6 @@ lazy val root = project.in(file("."))
   )
   .settings(BuildSettings.buildSettings)
   .settings(BuildSettings.publishSettings)
-  .settings(BuildSettings.formattingSettings)
```

Comment:
When is it run then?

Reply:
It is only checked in CI, so it's up to you, whether you use formatting on save in IDE, or via sbt.